tests/tape/tapes/test_qubit_param_shift.py

```diff
@@ -541,3 +541,38 @@ def test_expval_and_variance(self, tol):
         ).T
         assert gradA == pytest.approx(expected, abs=tol)
         assert gradF == pytest.approx(expected, abs=tol)
+
+class TestHessian:
+    """Tests for parameter Hessian method"""
+
+    def test_hessian_calculation(self, tol):
+        # example data taken from https://arxiv.org/pdf/2008.06517.pdf
+        n_wires = 5
+        dev = qml.device("default.qubit", wires=n_wires)
+        weights = [2.73943676, 0.16289932, 3.4536312 , 2.73521126, 2.6412488]
+
+        with QubitParamShiftTape() as tape:
+            for i in range(n_wires):
+                qml.RX(weights[i], wires=i)
+
+            qml.CNOT(wires=[0, 1])
+            qml.CNOT(wires=[2, 1])
+            qml.CNOT(wires=[3, 1])
+            qml.CNOT(wires=[4, 3])
+
+            qml.expval(qml.PauliZ(1))

```

Comment:
It might also be good to have a test that tests the Hessian computation for a vector-valued tape! e.g., one that has multiple expval returns, or a probability return.